packages/commonjs/README.md

```diff
@@ -0,0 +1,172 @@
+
+## Options
+
+### `exclude`
+
+Type: `String` | `Array[...String]`
+Default: `null`
+
+A [minimatch pattern](https://github.com/isaacs/minimatch), or array of patterns, which specifies the files in the build the plugin should _ignore_. By default non-CommonJS modules are ignored.
+
+### `include`
+
+Type: `String` | `Array(String)` \
+Default: `null`
+
+A [minimatch pattern](https://github.com/isaacs/minimatch), or array of patterns, which specifies the files in the build the plugin should operate on. By default CommonJS modules are targeted.
+
+### `extensions`
+
+Type: `Array(String)` \
+Default: `['.js']`
+
+Search for extensions other than .js in the order specified.
+
+### `ignoreGlobal`
+
+Type: `Boolean` \
+Default: `false`
+
+If true, uses of `global` won't be dealt with by this plugin.
+
+### `sourceMap`
+
+Type: `Boolean` \
+Default: `true`
+
+If false, skips source map generation for CommonJS modules.
+
+### `namedExports`
+
+Type: `Object` \
+Default: `null`
+
+Explicitly specify unresolvable named exports.
+
+This plugin will attempt to create named exports, where appropriate, so you can do this...
+
+```js
+// importer.js
+import { named } from './exporter.js';
+
+// exporter.js
+module.exports = { named: 42 }; // or `exports.named = 42;`
+```
+
+...but that's not always possible:
+
+```js
+// importer.js
+import { named } from 'my-lib';
+
+// my-lib.js
+var myLib = exports;
+myLib.named = "you can't see me";
+```
+
+In those cases, you can specify custom named exports:
+
+```js
+commonjs({
+  namedExports: {
+    // left-hand side can be an absolute path, a path
+    // relative to the current directory, or the name
+    // of a module in node_modules
+    'my-lib': ['named']
+  }
+});
+```
+
+### `ignore`
+
+Type: `Array(String | (String) => Boolean)` \
+Default: `[]`
+
+Sometimes you have to leave require statements unconverted. Pass an array containing the IDs or an `id => boolean` function. Only use this option if you know what you're doing!
+
+## Using with @rollup/plugin-node-resolve
+
+Since most CommonJS packages you are importing are probably depdenencies in `node_modules`, you may need to use [@rollup/plugin-node-resolve](https://github.com/rollup/plugins/packages/node-resolve):
+
+```js
+// rollup.config.js
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+
+export default {
+  input: 'main.js',
+  output: {
+    file: 'bundle.js',
+    format: 'iife',
+    name: 'MyModule'
+  },
+  plugins: [resolve(), commonjs()]
+};
+```
+
+## Usage with symlinks
+
+Symlinks are common in monorepos and are also created by the `npm link` command. Rollup with `rollup-plugin-node-resolve` resolves modules to their real paths by default. So `include` and `exclude` paths should handle real paths rather than symlinked paths (e.g. `../common/node_modules/**` instead of `node_modules/**`). You may also use a regular expression for `include` that works regardless of base path. Try this:
```

Comment:
```suggestion
Symlinks are common in monorepos and are also created by the `npm link` command. Rollup with `@rollup/plugin-node-resolve` resolves modules to their real paths by default. So `include` and `exclude` paths should handle real paths rather than symlinked paths (e.g. `../common/node_modules/**` instead of `node_modules/**`). You may also use a regular expression for `include` that works regardless of base path. Try this:
```